packages/cli/src/aws/update-server.ts

```diff
@@ -1,32 +1,52 @@
-import { UpdateServiceCommand } from '@aws-sdk/client-ecs';
-import { ecsClient, getEcsServiceName, getStackByTag } from './utils';
+import { getServerVersions } from './utils';
+import { configFileName, readConfig, writeConfig } from '../utils';
+import { MedplumInfraConfig } from '@medplum/core';
+import * as semver from 'semver';
+import { spawnSync } from 'child_process';
+import { createMedplumClient } from '../util/client';
 
 /**
  * The AWS "update-server" command updates the Medplum server in a Medplum CloudFormation stack.
  * @param tag - The Medplum stack tag.
+ * @param options - Client options
  */
-export async function updateServerCommand(tag: string): Promise<void> {
-  const details = await getStackByTag(tag);
-  if (!details) {
-    console.log('Stack not found');
+export async function updateServerCommand(tag: string, options: any): Promise<void> {
+  const client = await createMedplumClient(options);
+  const config = readConfig(tag) as MedplumInfraConfig;
+  if (!config) {
+    console.log(`Configuration file ${configFileName(tag)} not found`);
     return;
   }
-  const ecsCluster = details.ecsCluster?.PhysicalResourceId;
-  if (!ecsCluster) {
-    console.log('ECS Cluster not found');
-    return;
+  const separatorIndex = config.serverImage.lastIndexOf(':');
+  const initialVersion = config.serverImage.slice(separatorIndex + 1);
+  let updateVersion = await nextUpdateVersion(initialVersion);
+  while (updateVersion) {
+    console.log(`Performing update to v${updateVersion}`);
+    config.serverImage = `${config.serverImage.slice(0, separatorIndex)}:${updateVersion}`;
+    deployServerUpdate(tag, config);
+    // Run data migrations
+    await client.post('/admin/super/migrate', '');
```

Comment:
`/admin/super/migrate` is an async endpoint... I think we'll need to get the `AsyncJob` id from the response, and poll on it until it's done, yeah?